velox/docs/functions/presto/decimal.rst

```diff
@@ -106,31 +106,43 @@ implementation matches Presto.
 Division
 --------
 
-Perfect division is not possible. For example, 1 / 3 cannot be represented as a
-decimal value.
+Let’s say a is a decimal(p1, s1) with unscaled value A and b is a decimal(p2, s2) with unscaled value B.
```

Comment:
I'm not able to follow the new docs. Is there any particular reason you decided to re-write existing doc instead of appending to the end of it?

Reply:
> We know the reason why we set the scale to max(s1,s2) and wanted to explain it.

Interesting. What are these reasons? I didn't realize we understand these.

Reply:
> [docs] When dividing a number with p1 digits over a number of s2 scale, we need p1 + s2 digits for the result. In the worst case we divide by 0.0000001, which effectively is a multiplication by 10^s2.

>  [karteekmurthys] The sentence before this doesn't mention anything about 0.0000001, not sure why we mention this here.

Perhaps, we could edit this sentence like so:

```
When dividing a number with p1 digits over a number of s2 scale, the biggest result requires s2 extra digits before the decimal point. To get the largest number we must divide by 0.0000001, which effectively is a multiplication by 10^s2. Hence, precision of the result needs to be at least p1 + s2.
```

Reply:
> Interesting. What are these reasons? I didn't realize we understand these.

I have added an example in the PR. It helps in keeping the rescale factor in check.

Reply:
@mbasmanova Snapshot after generating html files.

<img width="822" alt="Screenshot 2024-04-08 at 10 17 28 AM" src="https://github.com/facebookincubator/velox/assets/6971561/a414e56b-7f03-4a97-8945-b24cd70f7d8e">

Reply:
`Let’s say a is a decimal(p1, s1)`.
Maybe use `number1` or `value1`?
Also, enclose under ` `` ` to highlight.